apps/wing/src/commands/test.ts

```diff
@@ -141,11 +144,11 @@ async function testOne(entrypoint: string, options: TestOptions) {
 
   switch (options.target) {
     case Target.SIM:
-      return await testSimulator(synthDir);
+      return await testSimulator(synthDir, options.clean);
     case Target.TF_AWS:
-      return await testTfAws(synthDir);
+      return await testTfAws(synthDir, options.clean);
     case Target.AWSCDK:
-      return await testAwsCdk(synthDir);
+      return await testAwsCdk(synthDir, options.clean);
```

Comment:
nit: I think these functions should take an entire `TestOptions` interface instead of just this one flag